### DISCUSSION TITLE
can I use Pool Contract Address and Farmer Key 1 for key 2 to build the plot?

### DISCUSSION THREAD

**Discussion by brightways021** [Chia-Network/chia-blockchain#8166] [General]
Hi. I have 2 private keys. As we know, we haver 1 Farmer Key and 1 Pool Contract Address for each key. We know that to build each plot, we must use the Farmer Key and Pool Contract Address for each key. Now the question is, can I use Pool Contract Address and Farmer Key 1 for key 2 to build the plot? If the answer is yes, does this not cause a problem?

**Comment by BrandtH22** [comment]
@brightways021 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

Note - it is recommended to use the pool contract address and the farmer key from the same keyset to avoid any potential issues with claiming rewards. These rewards can be sent to a different keysets address by changing the payout address information for both the farmer payout and pool payout.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
